internal/db/postgres_test.go

```diff
@@ -24,7 +24,9 @@ func TestConnectToDB(t *testing.T) {
 
 func TestConnectionPoolConfiguration(t *testing.T) {
 	pm := PostgresManager{}
-	os.Setenv("MPS_DB_MAX_OPEN_CONNS", "7")
+	if err := os.Setenv("MPS_DB_MAX_OPEN_CONNS", "7"); err != nil {
+		t.Fatalf("Failed to set environment variable: %v", err)
```

Comment:
lets stick with using the assert package.

Reply:
done